.github/settings.yml

```diff
@@ -55,7 +55,7 @@ branches:
         # Required. Require branches to be up to date before merging.
         strict: true
         # Required. The list of status checks to require in order to merge into this branch
-        contexts: ["test (11.x)", "test (12.x)", "test (14.x)", "license/cla"]
+        contexts: ["test ([abstractNodeVersion])", "license/cla"]
```

Comment:
`[abstractNodeVersion]` is a template variable. This needs to be updated with a version of Node to test. E.g. `18.x`.